compiler/src/main/java/motif/compiler/codegen/JavaPoetUtil.kt

```diff
@@ -143,24 +143,32 @@ interface JavaPoetUtil : ParserUtil {
         return NameScope(env).block()
     }
 
+    fun RequiredDependency.parameterSpec(name: String): ParameterSpec {
```

Comment:
> Do not include qualifiers on overidden child dependencies methods

why not?

Reply:
It seemed unnecessary. This is pure implementation details and java doesn't require copying over annotations of overridden methods.

Before:
```java
class FooScopeImpl {

  @Override
  public BarScope bar() {
    return new BarScopeImpl(new BarScopeImpl.Dependencies() {

      @ActivityContext
      @Override
      public Context context() {
        return component.activityContext();
      }
    });
  }

  ...
}
```

After:
Before:
```java
class FooScopeImpl {

  @Override
  public BarScope bar() {
    return new BarScopeImpl(new BarScopeImpl.Dependencies() {

      // No annotation here
      @Override
      public Context context() {
        return component.activityContext();
      }
    });
  }

  ...
}
```

Reply:
sounds good